File: MyLittlePony/Player.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MyLittlePony
{
    class Player
    {
        public int _id;
        public string _name;
        private List<Card> _cards = new List<Card>();

        public Player(string name, int ID)
        {
            this.setName(name);
            this.setID(ID);
        }

        public string getName()
        {
            return this._name;
        }

        public void setName(string name)
        {
            this._name = name;
        }

        public int getID()
        {
            return this._id;
        }

        public void setID(int ID)
        {
            this._id = ID;
        }

        public List<Card> getCards()
        {
            return this._cards;
        }

        public void playCard()
        {
            this.getCurrentCard();
            //this.chooseProperty();
        }

        public Card getCurrentCard()
        {
            Card topCardOfCurrentPlayer = this._cards.First();
            return topCardOfCurrentPlayer;
        }

        public Property chooseProperty()
        {
            List<Property> propertiesOfTopCardOfCurrentPlayer = new List<Property>();

            propertiesOfTopCardOfCurrentPlayer = this.getCurrentCard().getProperties();

            Console.WriteLine("Hallo " + this.getCurrentCard().getPlayer().getName());
            Console.WriteLine("Deine oberste Karte:\n");
            Console.WriteLine("Name: " + this.getCurrentCard().getName());
            Console.WriteLine("Eigenschaften:\n");

            for (int i = 0; i <= 3; i++)
            {
                Console.WriteLine("[" + i + "] " + propertiesOfTopCardOfCurrentPlayer[i].getValue() + " " + propertiesOfTopCardOfCurrentPlayer[i].getUnit());
            }

            Console.WriteLine("Wähle eine Property aus.");
            int propertyIndex = Convert.ToInt32(Console.ReadLine());

            Property choosenProperty = this.getCurrentCard().getProperties()[propertyIndex];

            return choosenProperty;
        }

        public void countCards()
        {

        }
    }
}

File: Classes/Domain/Model/Player.php
<?php
declare(strict_types=1);

namespace robertontl\mylittlepony\Classes\Domain\Model;

class Player
{
    /**
     * @var int
     */
    protected $id;

    /**
     * @var string
     */
    protected $name = '';

    /**
     * @var array
     */
    protected $cards = array();

    /**
     * @var array
     */
    protected $trumpedCards = array();

    public function setId(int $id): void
    {
        $this->id = $id;
    }

    public function getId(): int
    {
        return $this->id;
    }

    public function setName(string $name): void
    {
        $this->name = $name;
    }

    public function getName(): string
    {
        return $this->name;
    }

    public function setCards($card): void
    {
        array_push($this->cards, $card);
    }

    public function getCards(): array
    {
        return $this->cards;
    }

    public function setCardsAfterBattle(array $playerCards): void
    {
        $this->cards = array();

        $this->cards = $playerCards;
    }

    public function chooseProperty($property): int
    {
        return $this->cards[0]->getProperties()[$property];
    }

    public function setTrumpedCards($card, $playerId): void
    {
        $card->setPlayerId($playerId);
        $card->setTrumped(true);
        array_push($this->trumpedCards, $card);
    }

    public function getTrumpedCards(): array
    {
        return $this->trumpedCards;
    }
}
